src/components/apis/details-of-api/detailsOfApiHandlers.ts

```diff
@@ -14,4 +18,31 @@ export class DetailsOfApiHandlers implements IWidgetHandler {
 
         return widgetOrder;
     }
+    public getStyleDefinitions(): StyleDefinition {
+        return {
+            colors: {
+                badgeColor: {
+                    displayName: "API type badge color",
+                    defaults: {
+                        value: "#636363"
```

Comment:
are there any global color settings? Or it is a code convention to write down color-values explicitly in every component?

Reply:
There are global color settings. By defining the missing colors for components like this, they will be added dynamically in the global definitions.

The reason for this is that the global colors are defined in the default.json (which is the default stylesheet for the developer portal, before users make and save any customizations). 
Adding them there is not possible, since it would require users to reset their developer portal in order to get the new global stylesheet and have those changes.

Reply:
got it, thanks!